client/src/pages/ChatPage.js

```diff
@@ -192,23 +202,42 @@ const ChatPage = () => {
 
         <div className="flex justify-center">
           <div className="mb-5 absolute bottom-0 w-3/5">
-            <Card className="flex mb-5">
-              <ul className="p-5 max-h-[30rem] overflow-y-auto scrollbar-width:none">
-                {chat.history
-                  ? chat.history.map((message, index) => (
-                      <li key={index}>
-                        <span className="font-bold">{message.username}: </span>
-                        {message.message}
-                      </li>
-                    ))
-                  : null}
-              </ul>
+            <Card className="flex mb-5 px-5 py-0 max-h-[30rem] overflow-y-auto scrollbar scrollbar-none shadow-none">
+              {currChat.history &&
```

Comment:
![image](https://github.com/awctw/study-dash/assets/56493743/82b2c554-0f3e-4c3b-b261-8490b69a2631)

Chat history does not go to top of page (you can see the next bubble is cut off at the top) and there is no scroll bar for chat history in case it is long?

Another future enhancement could also be to load long chat histories pieces at a time.